mm2src/lp_native_dex.rs

```diff
@@ -344,6 +344,11 @@ pub async fn lp_init(ctx: MmArc) -> Result<(), String> {
 
     spawn(broadcast_maker_orders_keep_alive_loop(ctx.clone()));
 
+    let i_am_seed = ctx.conf["i_am_seed"].as_bool().unwrap_or(false);
+    if i_am_seed {
```

Comment:
It seems that leak is actual for non-seed nodes too - but it is less noticeable. Could you please start `clean_memory_loop` unconditionally?

Reply:
Yeah, I guess you are right. I also still have some minor fixes to add to this PR plus investigating if the "light" peers memory leak is fixed https://github.com/KomodoPlatform/atomicDEX-API/issues/941#issuecomment-896000408